releasenotes/notes/ad-podman-166025a85e86f2e8.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Autodiscovery of integrations now works with Podman containers.
```

Comment:
Do we have a minimum required Podman version?

Reply:
This feature relies on the functions that we copied from the podman client here: https://github.com/DataDog/datadog-agent/blob/8495634aab12d2feed53703eafd1172b70836ac4/pkg/util/podman/db_client.go#L22

This means that we support the same as the 3.4.1 version of the client does. I can try to find more info to be more specific in the release notes.

Reply:
I tried it with 3.0.0 (latest major version is 3) and it works fine.
I added it to the release notes.